doc/developers/testsuite.html

```diff
@@ -292,6 +292,33 @@ <h3>How to interpret the output</h3>
     </p>
 
 
+    <a name="memcheck"></a>
+     <h3>Running with valgrind</h3>
+
+     <p>
+       The testsuite can also be run using <a href="http://valgrind.org/">valgrind</a>
+       to check for <i>memory corruption</i> in the library.
+     </p>
+
+     <p>
+       You can do so by invoking
+ <pre>
+ ctest -DMEMORYCHECK=ON <...> -S ../tests/run_testsuite.cmake
+ </pre>
+       when running the testsuite, or directly by
+ <pre>
+ ctest <...> -S ../tests/run_memorycheck.cmake
```

Comment:
@masterleinad Oh, just spotted that: Can you create a new pr that checks in the `run_memorycheck` file, please? :-)

Reply:
Of course, see #6041.